packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -1544,7 +1544,12 @@ abstract class FlutterCommand extends Command<void> {
   int? intArg(String name) => argResults?[name] as int?;
 
   /// Gets the parsed command-line option named [name] as `List<String>`.
-  List<String> stringsArg(String name) => argResults?[name] as List<String>? ?? <String>[];
+  List<String> stringsArg(String name) {
```

Comment:
After this issue https://github.com/flutter/flutter/issues/105263#issuecomment-1145435487 we are going to change this to bubble up the error when the flag is missing so the behavior we are looking here will be something like `arResults?[name]! as List<String>? ?? <String>[]`

Reply:
Sure, Changed to `argResults![name]! as List<String>? ?? <String>[];`

Reply:
Did not change it back to arrow operator, based upon [styleguide section on arrow operators](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#consider-using--for-short-functions-and-methods).

Reply:
LGTM